cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/VmTypeMeta.java

```diff
@@ -200,6 +207,11 @@ public VmTypeMetaBuilder withPrice(Double price) {
             return this;
         }
 
+        public VmTypeMetaBuilder withResourceDisk(Boolean resourceDisk) {
```

Comment:
however the hasResourcedisk, getHasResourceDisk is strange, but you need to use here the withHasresourceDisk as well